src/skills-builder/skills-builder-modal/skillsBuilderModal.scss

```diff
@@ -0,0 +1,9 @@
+.skills-builder-modal {
+  .pgn__modal-close-button{
+    color: #ffffff !important;
+  }
+
+  .pgn__modal-hero-bg {
+    background-color: #00262B;
+  }
```

Comment:
We generally don't want to modify the classes prefixed with `pgn__`. I'd recommend using utility classes for these instead, which would also eliminate the need for the SCSS file

Reply:
Two things here:
-- I needed to override the color of the close button, which is part of the Paragon component. This is how it was done in the B2B version. How would I override the color of the button otherwise?
-- The background color of the hero doesn't exist as a utility class but this is the color used in the Figma mocks. What do I do here then?

Reply:
1. Good point! It appears we don't have a prop in the `ModalDialog` for modifying the color of the icon. You can definitely do it this way and it won't cause any issues. I think it's more of a nit at this point, but I would go for something like:
```
.skills-builder-modal {
  button[aria-label="Close"][type="button"]{
    color: #ffffff;
  }
}
```

I personally am not a huge fan of using the `!important` tag unless we absolutely have to. The above method gets around it by having higher specificity than the current styles.

2. You can add a `className` to the `ModalDialog.Hero.Background` component and give it a utility class `bg-primary-500` and that should take care of that without having to do any custom SCSS.

FWIW, this is mostly a product of a discussion that has gone on in Frontend groups. The consensus is that our design system _should_ be able to do all of this without any custom styling. So the fact that we need to do this is a sign that perhaps the system has room for improvement in this area

Reply:
1. Your suggestion for the first point worked great. Thanks for the help here, even though I do find this harder to understand than the above solution (to me). 
2. This is super straight forward too. Thanks.